pkg/services/libraryelements/database.go

```diff
@@ -459,9 +495,10 @@ func (l *LibraryElementService) getAllLibraryElements(c context.Context, signedI
 		builder.Write(selectLibraryElementDTOWithMeta)
 		builder.Write(", le.folder_uid as folder_uid ")
 		builder.Write(getFromLibraryElementDTOWithMeta(l.SQLStore.GetDialect()))
+		builder.Write(" LEFT JOIN folder f ON le.folder_uid = f.uid AND le.org_id = f.org_id")
```

Comment:
This is not needed anymore, we can filter directly by `folder_uid` and `org_id` in the `library_element` table